ncm-opennebula/src/main/resources/datastore.tt

```diff
@@ -1,23 +1,26 @@
 [%- lists = ['bridge_list', 'ceph_host'] -%]
 [%- integer = ['rbd_format'] -%]
 [%- booleans = ['datastore_capacity_check'] -%]
-[%- FOR pair IN datastore.pairs -%]
+[%- FOR pool IN datastore.pairs -%]
+NAME = "[% pool.key %]"
+[%    FOR pair IN pool.value.pairs -%]
 [% # If we are installing a Ceph datastore we should remove
    # ceph_user_key from ONE template to avoid any sec issue-%]
-[%      IF pair.key == 'ceph_user_key' -%]
-[%           -%]
-[%      ELSE -%]
-[%          pair.key FILTER upper %] =
-[%-             SWITCH pair.key -%]
-[%                  CASE lists -%]
- "[%                    pair.value.join(' ') %]"
-[%                  CASE booleans -%]
- "[%                    pair.value ? "yes" : "no" %]"
-[%                  CASE integer -%]
- [%                     pair.value %]
-[%                  CASE -%]
- "[%                    pair.value %]"
-[%              END -%]
-[%      END -%]
+[%        IF pair.key == 'ceph_user_key' -%]
+[%             -%]
```

Comment:
`ceph_user_key` is not included in the template for security reasons, only the Ceph UUID, but now reading the tt file it makes more sense to include this as a `SWITCH` case as we do for the users... I will change that